docs/docs/sample-scripts/spo/list-tenant-alert-usage/index.mdx

```diff
@@ -0,0 +1,159 @@
+---
+tags:
+  - reports
+  - alerts
+  - retirement
+  - tenant
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# List SharePoint alerts usage across the tenant
+
+Author: [Saurabh Tripathi](https://github.com/saurabh7019)
+
+SharePoint Online list alerts are being gradually retired. This script scans all sites across the tenant and generates a comprehensive CSV report of existing alerts. This information helps administrators identify and plan the migration of critical alerts to modern alternatives.
+
+- Prerequisites: This script requires app-only permissions with full control on all site collections to run.
+
+<Tabs>
+  <TabItem value="PowerShell">
+
+  ```powershell
+  $fileExportPath = "Alerts.csv"
+
+  function Add-AlertsToResults {
```

Comment:
This function is called `Add-...` while it doesn't really add anything. Instead it transforms data to another structure. Therefore I think starting the function with `Get-` would be clearer.